### PR TITLE
Formatted and trimmed trailing whitespace in deployscript

### PR DESCRIPTION
**Pull request by picccard**
Makes the deployscript more readable by formating and trimming trailing whitespace with vscode

### CODE REVIEW

**Comment by DCMattyG** [comment]
Hi there @picccard, in this project we are using 4 spaces are equivalent to a "TAB", and the whitespace you're removing is there on purpose for readability.....is this causing any technical issues that you've seen?

**Comment by DCMattyG** [comment]
Thanks for this @picccard, I agree this makes it more readable. I'll also add some formatting rules into the project that better enforce this where possible.

Appreciate the cleanup!

**Comment by DCMattyG** [comment]
Appreciate the formatting @picccard, my apologies for the mistake in my now deleted comment....

What I mean to say was that for PowerShell files, I currently have it set to using 2x Spaces as a TAB as when using 4x Spaces it looks a bit too clunky. I have the same settings applied for JavaScript as well which follows a similar format.

For Python, we use 4x Spaces as a TAB since Python is indentation-sensitive, whereas PowerShell and JavaScript are not.

Does that make sense and/or do you agree?

TL:DR - For languages that aren't sensitive to indentation, 2x Spaces / TAB looks a bit nicer and has less empty whitespace.

**Comment by DCMattyG** [comment]
Thank you so much @picccard, appreciate you!

**Comment by picccard** [comment]
Python code for the engine is using 4 spaces and javascript code for the ui is using 2 spaces.

Changed the deployscript and updatescript to use 2 spaces.

Perhaps the project should have some contributing guidelines? Example from [ALZ-Bicep Contributing](https://github.com/Azure/ALZ-Bicep/wiki/Contributing)

**Comment by DCMattyG** [comment]
That's a great point @picccard, I'll definitely work on that.

Haven't had many contributions in the last few years and all of the sudden I have more than I can handle hahaha! Good problem to have.

**Comment by DCMattyG** [comment]
Hey @picccard, there were a lot of updates in the recent release and this PR will need those changes merged in before we can proceed. Thanks so much!
